pkg/gadgets/top/ebpf/tracer/tracer.go

```diff
@@ -334,6 +334,7 @@ func (t *Tracer) nextStats() ([]*types.Stats, error) {
 			CumulativeRunCount: cumulativeRunCount,
 			MapMemory:          totalMapMemory,
 			MapCount:           uint32(len(mapIDs)),
+			CpuUsage:           100 * float64(curRuntime) / float64(t.config.Interval.Nanoseconds()),
```

Comment:
should this be divided by the number of online CPUs to avoid it being > 100% as you mentioned in the commit message?

Reply:
The motivation is to compare the cpu usage of ebpf progs to the cpu usage shown in linux `top` command. So we adopt the same way as what they did in linux `top` command. linux `top` also summed up cpu overheads in all cpu cores, it can be >100% as well.

Reply:
Make sense.

Reply:
Hmm, I'm not too sure about this. I get that this is what top also does, but top only ever works on one node. If you have a cluster with different hardware for the nodes, this would make it even trickier to compare and find bottlenecks, right? Maybe we should at least provide another value where the time is divided by the number of cores?

Reply:
What could be the name of that second value?

Reply:
> Hmm, I'm not too sure about this. I get that this is what top also does, but top only ever works on one node. If you have a cluster with different hardware for the nodes, this would make it even trickier to compare and find bottlenecks, right? Maybe we should at least provide another value where the time is divided by the number of cores?

Yeah, your concerns make sense. I agree maybe we can provide both two metrics.

> What could be the name of that second value?

maybe `avg_cpu_utilization` or `avg_cpu_util`?

Reply:
What about having `TotalCpuUsage` and `PerCpuUsage`? (and adding the correct documentation to explain what they are about)